ajuna-test/src/traits.rs

```diff
@@ -0,0 +1,67 @@
+use frame_system::Config;
+use sp_io::TestExternalities;
+use sp_runtime::{
+	traits::{AtLeast32BitUnsigned, One},
+	Storage,
+};
+
+pub trait BlockNumbers<B> {
+	fn block_number() -> B;
+	fn set_block_number(n: B);
+	fn current_block_number() -> B;
+}
+
+pub trait RuntimeBuilding<Runtime: Config, B: AtLeast32BitUnsigned + One, T: BlockNumbers<B>> {
+	fn configure_storages(&self, _storage: &mut Storage);
```

Comment:
For me, singular `configure_storage` makes more sense with single `Storage` struct